### PR TITLE
Add new graphs for environmental performance

### PR DESCRIPTION
**Pull request by guilbep**
# Describe your changes
##  Feature concerning the Shelter app
- Add Sankey and Sunburst type of graph for Environmental performance screen

## Related GitHub issues and pull requests

- Ref: #131 

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] Don't forget to link PR to issue if you are solving one.

### CODE REVIEW

File: frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue [hunk 1]
```diff
@@ -35,83 +51,368 @@ export default class GraphTree extends Vue {
...
+      return this.optionTreeMap;
+    }
+    if (this.graphType === "sankey") {
+      return this.optionSankey;
+    }
+    throw new Error("graphType does not exist");
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public tooltipFormatter(params: any): string {
```
Comment (blueur) [inline frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue:276]: try CallbackDataParams
import { CallbackDataParams } from "echarts/types/dist/shared";

File: frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue [hunk 2]
```diff
@@ -35,83 +51,368 @@ export default class GraphTree extends Vue {
...
-            return "error: format number undefined";
-          }
-          const name =
-            params.treeAncestors[params.treeAncestors.length - 1].name;
-          const v = params.data.value[0];
-          const unit = params.data.value[1];
-          return `</div>${name}: ${formatNumber(v)} ${unit}</div>`;
+      trigger: "item",
+      confine: false,
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (blueur) [inline frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue:305]: eslint disable needed ?

File: frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue [hunk 3]
```diff
@@ -35,83 +51,368 @@ export default class GraphTree extends Vue {
...
+      title: this.titleOption,
+      series,
+    };
+  }
 }
 interface datatree {
   name: string;
   value: number[];
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  itemStyle: any;
```
Comment (blueur) [inline frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue:400]: Record<string, string> ?

**Review by blueur — APPROVED** [review]

File: frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue (hunk 1 above)
Comment (guilbep) [reply frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue:191]: @blueur ça fonctionne pas car Property 'treePathInfo' does not exist on type 'CallbackDataParams'.Vetur(2339) et pareil pour value et data. Si t'as une plus belle solution je suis preneur

File: frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue (hunk 2 above)
Comment (guilbep) [reply frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue:220]: bien vue @blueur

File: frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue (hunk 3 above)
Comment (guilbep) [reply frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue:315]: Ça fonctionne @blueur merci

File: frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue (hunk 1 above)
Comment (blueur) [reply frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue:184]: Tu peux aussi étendre CallbackDataParams

Comment (guilbep) [reply frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue:184]: I tried, and I failed. Thank you echarts

**Pull request merged**